code/parachain/runtime/picasso/src/lib.rs

```diff
@@ -715,7 +715,7 @@ impl Contains<Call> for BaseCallFilter {
 
 impl call_filter::Config for Runtime {
 	type Event = Event;
-	type UpdateOrigin = EnsureRootOrOneThirdNativeTechnical;
+	type UpdateOrigin = EnsureRootOrTwoThirdNativeTechnical;
```

Comment:
Do we want the technical council to also be 2/3 majority voting? That may slow down emergency action no?

Reply:
i do not think this change is good.